### ISSUE TITLE
Tensorflow or TFLite providers usage on PaspberryPi

### ISSUE THREAD

**Issue by posuhov** [deepjavalibrary/djl#2773]
Hi team, 
Your releases include only compiled libraries for windows and linux platforms. Could you please advise how to use them on arm processors like Raspberry Pi? Is there any guide or recommendations how to compile needed libraries for ARM architecture?

**Comment by frankfliu** [comment]
@posuhov 

We don't have plan to add arm support for FTLite engine. Here is a few options:
1. consider use onnxruntime or PyTorch model for ARM
2. You can build your own TFLite binary from source, then you package arm binary in DJL's jar or cache folder. DJL has all necessary logic to load the .so file.

**Comment by posuhov** [comment]
@frankfliu  thanks for pointing me out on possible solutions. I will post my current results here: I was not able to deal with PyTorch - their models have awful documentation, I spent days on trying to figure out what the model returns and how to use these results but without any luck. Then I've tried to build tensorflow 2.10.1 on raspberry 3B - after 2 weeks of compilation I have interrupted a process. Then I've compiled same on Ubuntu and Core i7 in 3 hours, but I wasn't able to use it together with DJL - it couldn't load .so file because of linking errors. Finally I've implemented object detection with java and plain tensorflow 2.10.1 but it's quite slow. I'm going to build TFLite for raspberry and try if it will be better
